source/atlas-cli-deploy-local.txt

```diff
@@ -86,6 +86,105 @@ Before you begin, complete the following prerequisites:
 
 .. include:: /includes/steps-atlas-cli-deploy-local-prereqs.rst
 
+.. _atlas-cli-pvt-registry-setup:
+
+(Optional) Set up a Private Image Registry 
+------------------------------------------
+
```

Comment:
It might be useful to provide some context to readers who don't understand why you might choose to do this, something like:

> The Atlas CLI pulls a Docker image and creates a container in order to deploy the local cluster. By default, the Atlas CLI pulls from the public Docker Hub repo mongodb/mongodb-atlas-local. In some cases it is preferred to pull a specific image from an internal (often private) registry. You can configure the Atlas CLI to support this.